### PR TITLE
HELM-209: Datasource doc expansion

### PR DESCRIPTION
**Pull request by mmahacek**
# Pull Request Check Sheet

* [X] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [X] Have you made a comment in that issue which points back to this PR?
* [X] Have you updated the JIRA link at the bottom of this comment to link to your issue?

# Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

Once the reviewer(s) accept the PR and the branch passes continuous integration in Bamboo, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

# External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-290
* Continuous Integration: [CircleCI](https://circleci.com/gh/OpenNMS/opennms-helm)

### CODE REVIEW

**Comment by mmahacek** [comment]
There were some inconsistencies with how the different queries were notated between the three datasources.  This brings them closer into alignment and makes it easier to see what some of the attributes do at a quicker glance.

**Comment by mmahacek** [comment]
As a separate topic, Grafana changed their terminology from "Template Variable" to just "Variable" a few versions back, though we still have the old term used in the docs.  Might be a separate issue/discussion to look at updating our terminology for that.

**Review by Bonrob2 — APPROVED** [review]
I will assume that this is correct. :-)

File: docs/modules/datasources/pages/performance_datasource.adoc [hunk 1]
```diff
@@ -92,6 +101,15 @@ To query only SNMP interface resources on the node with ID 42, and to display th
 nodeResources(42, label, interfaceSnmp)
 ```
 
+The available arguments are as follows:
+[options="header, %autowidth"]
+|===
+| Name              | Required | Default  | Description
+| resourceId        | Required | `(none)` | The ID of the node (either databaseId or foreignSource:foreignId) for which to display resources.
+| textProperty      | Optional | `id`     | One of `id`, `label`, or `name` to display alternate string values in the variable drop down.
```
Comment (Bonrob2) [inline docs/modules/datasources/pages/performance_datasource.adoc:109]:
```suggestion
| textProperty      | Optional | `id`     | One of `id`, `label`, or `name` to display alternate string values in the variable drop-down menu.
```

**Review by Bonrob2 — COMMENTED** [review]
LGTM! Just one change.

**Review by Bonrob2 — APPROVED** [review]

**Review by Bonrob2 — APPROVED** [review]
LGTM! Thanks for updating!

**Pull request merged**